framework/src/main/java/org/tron/core/services/http/Util.java

```diff
@@ -439,4 +441,27 @@ public static void printAccount(Account reply, HttpServletResponse response, Boo
     }
   }
 
+  public static byte[] getAddress(HttpServletRequest request)
+          throws Exception {
+    byte[] address = null;
+    String addressParam = "address";
+    String addressStr = request.getParameter(addressParam);
+    if (org.apache.commons.lang3.StringUtils.isBlank(addressStr)) {
+      String input = request.getReader().lines()
+              .collect(Collectors.joining(System.lineSeparator()));
+      Util.checkBodySize(input);
+      JSONObject jsonObject = JSONObject.parseObject(input);
+      addressStr = jsonObject.getString(addressParam);
+    }
+    if (org.apache.commons.lang3.StringUtils.isNotBlank(addressStr)) {
```

Comment:
Why use "org.apache.commons.lang3.StringUtils" instead of "StringUtils"?

Reply:
it conflicts with StringUtils already used in Util.java

Reply:
you should not use org.apache.commons.lang3.StringUtils directly. Import the org.apache.commons.lang3.StringUtils. And remove the useless import files in "GetBrokerageServlet.java" and "GetRewardServlet.java"

Reply:
Thanks for your advice, will do.